<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectManager</class>
 <widget class="QDockWidget" name="ProjectManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Manager</string>
  </property>
  <widget class="QWidget" name="contents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeView" name="view">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCreateFolder">
   <property name="icon">
    <iconset resource="../data/Resources.qrc">
     <normaloff>:/icons/project_management/create_folder.png</normaloff>:/icons/project_management/create_folder.png</iconset>
   </property>
   <property name="text">
    <string>Create folder</string>
   </property>
   <property name="toolTip">
    <string>Create folder</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="../data/Resources.qrc">
     <normaloff>:/icons/project_management/add_new_file.png</normaloff>:/icons/project_management/add_new_file.png</iconset>
   </property>
   <property name="text">
    <string>Add new file</string>
   </property>
   <property name="toolTip">
    <string>Add new file</string>
   </property>
  </action>
  <action name="actionExistingFiles">
   <property name="icon">
    <iconset resource="../data/Resources.qrc">
     <normaloff>:/icons/project_management/add_existing_file.png</normaloff>:/icons/project_management/add_existing_file.png</iconset>
   </property>
   <property name="text">
    <string>Add existing file(s)...</string>
   </property>
   <property name="toolTip">
    <string>Add existing files</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../data/Resources.qrc">
     <normaloff>:/icons/project_management/rename.png</normaloff>:/icons/project_management/rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Rename file or folder</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../data/Resources.qrc">
     <normaloff>:/icons/project_management/remove.png</normaloff>:/icons/project_management/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected item(s) from the project</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../data/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
